### PR TITLE
refactor: remove unnecessary checks

### PR DESCRIPTION
**Pull request by DanielRuf**
**Summary**

There are two checks which are not needed. One check in a condition was added when the Facebook headers were removed and there is no reasons for the `false` check.

**Test plan**

### CODE REVIEW

**Comment by DanielRuf** [comment]
I guess the Git resolver change is related to https://github.com/yarnpkg/yarn/pull/3449.

**Comment by DanielRuf** [comment]
https://dev.azure.com/yarnpkg/yarn/_build/results?buildId=294&view=logs&jobId=2ff3c9c9-dec7-575d-2cca-4faaac369f41&taskId=ca90fcfb-b6eb-5ba5-e914-29a8da89a401&lineStart=169&lineEnd=169&colStart=34&colEnd=131

**Comment by DanielRuf** [comment]
Why do the CI checks fail?

**Comment by arcanis** [comment]
Restarting, but they looked pretty scary. Maybe related to the registry failures from past week?

File: src/cli/commands/check.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ export async function verifyTreeCheck(
         const relative = path.relative(cwd, subDepPath);
         const locations = path.normalize(relative).split(registry.folder + path.sep).filter(dir => !!dir);
         locations.pop();
-        while (locations.length >= 0) {
+        while (locations.length) {
```
Comment (arcanis) [inline src/cli/commands/check.js]: After giving it a second look this is incorrect - `>=` returns true on an empty array, but your change returns false. I think that's why the tests are failing 🙂

Comment (DanielRuf) [reply src/cli/commands/check.js]: Ok, I'll revert this one.

**Comment by arcanis** [comment]
Thanks 👍

**Pull request merged**